entity-framework/ef6/entity-framework-ef-documentation.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Entity Framework (EF) Documentation - EF6"
+title: "Get Started - EF6"
```

Comment:
I'd recommend you to have the words Entity Framework here too to help with SEO

Reply:
Will add it, but I am thinking about using the contents of this page as the basis to organize the ToC, at which point this page may just disappear. It is a bit strange because on MSDN this was the “documentation” root, and there was a level above that contained things that were not considered documentation (e.g. where to download), so I am going to do a little bit of refactoring.

Reply:
Sure. Also some of the SEO work can be done later as well.